config/supervisor/newapp.conf

```diff
@@ -1,6 +1,6 @@
 [program:newapp]
 command=/usr/bin/node /home/ubuntu/newapp/src/api/index.js
-process_name=%(program_name)s
+process_name=%(program_name)s --require=reify --require=async-to-gen/register
```

Comment:
When launching node arg list must be `--require reify` instead of `--require=reify` (with mocha `=` works but not with node, if I can remember)